### PR TITLE
Input

### PR DESCRIPTION
**Pull request by lannisterrr**
https://user-images.githubusercontent.com/29068538/152315715-252512f1-b51d-4865-8ca2-626ffb48fe8a.mp4

1.) Added cutom text area with extra styles for the placeholder
2.) Added custom input area for email 
3.) Added custom radio button with animation and my primary color.
4.) Added custom check box with animation and my primary color.
5.) Added custom range slider that is responsive every browser

### CODE REVIEW

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me.

**Review by piyushsh25 — COMMENTED** [review]
Looks good to me. 👍

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  display: inline-flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.radio__input {
+  display: none;
+}
+
+.radio__radio {
```
Comment (RakshithBhat03) [inline components/input/input.css:49]: This class can have a better name. Everything else looks good

Comment (lannisterrr) [reply components/input/input.css:49]: I am using BEM convention here.

Comment (RakshithBhat03) [reply components/input/input.css:49]: Okay, but use it consistently throughout the CL👍

**Pull request merged**